cpp/arcticdb/CMakeLists.txt

```diff
@@ -3,52 +3,73 @@ option(SSL_LINK "Link against SSL libraries." ON)
 option(STRIP_BUILDS "Strip built targets of all symbols" OFF)
 option(HIDE_LINKED_SYMBOLS "Passed `exclude-libs` through to linker, ensuring linked library symbols are not exported. Shrinks binary size." ON)
 option(STATIC_LINK_STD_LIB "Statically link the C++ standard libraries" ON)
-option(EXTERNAL_RELEASE "Set any compiler options needed for external releases" OFF)
 
-find_package(GTest REQUIRED)
+find_package(GTest CONFIG REQUIRED)
 
 include(CTest)
 include(GoogleTest)
 include(GenerateExportHeader)
 
-find_package(Threads REQUIRED)
-find_package(Zstd REQUIRED)
-find_package(LZ4 REQUIRED)
+# System and third-party libraries with built-in find_package Config
+cmake_policy(PUSH)
+    if (EXISTS "/usr/local/lib64/aws-c-cal/cmake/modules/FindLibCrypto.cmake") # Workaround old AWS SDK bug
+        cmake_policy(SET CMP0045 OLD)
```

Comment:
The PUSH and POP is quite fragile considering we may not necessarily set a policy at all?
Why do we need this, we'll get the AWS lib from VCPKG each time rather than relying on a system install? Does the VCPKG version still have this bug?

Oh - this is for compat with the internal `man.arcticdb` build?

Reply:
Yes. Newer AWS SDK seems to be fine.

Reply:
Why don't we leave this out then? We won't need `man.arcticdb` once this is in.

Reply:
It still requires some effort to get vcpkg to download reliably and similarly for a new dev image.

If it only takes a few lines to keep the old builds working, I would rather do that.